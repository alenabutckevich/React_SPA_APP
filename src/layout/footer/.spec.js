import React from 'react';
import Footer from './';

it('should render footer correctly', () => {
    const wrapper = shallow(
      <Footer/>
    );
    expect(wrapper).toMatchSnapshot();
});
