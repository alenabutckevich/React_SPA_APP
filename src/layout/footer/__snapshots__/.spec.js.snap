// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> render footer 1`] = `
<div
  className="footer"
>
  <Logo />
</div>
`;
